### ISSUE TITLE
Tag release v1.5.4 on github

### ISSUE THREAD

**Issue by yorickvP** [tomerfiliba/reedsolomon#28]
Hi! I'm packaging this module for NixOS. It seems that the pypi tarball does not include the `tests` dir, so I'm using the source from the github repo. It would be useful if the latest release, v1.5.4 were tagged here.
```bash
git tag v1.5.4 73926cdf81b39009bd6e46c8d49f3bbc0eaad4e4
git push origin v1.5.4
```

**Comment by lrq3000** [comment]
Hello, it's because v1.5.4 has not been officially released yet. I'm going to roll it out now.

**Comment by lrq3000** [comment]
Ok v1.5.4 is now released on github too, you can use it.

(/CORRECTION of my previous message: since this was only a pip fix, v1.5.4 was only released to pypi, but now the incongruency with github releases is fixed.)

**Comment by lrq3000** [comment]
I close for now, please let me know if there is still an issue and I'll reopen the ticket. Thank you for opening this issue!
